### DISCUSSION TITLE
Disable Allow Merge Commits

### DISCUSSION THREAD

**Discussion by ajeckmans** [reqnroll/Reqnroll#107] [Contributor Q&A] [answered]
I accidentally merged a PR again...

Should we disable 'Allow Merge Commits' and 'Allow rebase merging' to prevent this from happening in the future?

**Comment by gasparnagy** [comment]
@ajeckmans Yes. We should. Let me check that.

**Reply by gasparnagy** [reply, marked as answer]
Done. This should be enforced now along with some other settings (same what we have for Reqnroll.VisualStudio):

* Require a pull request before merging (Require all commits be made to a non-target branch and submitted via a pull request before they can be merged.) with 1 required approval minimum.
* Block force pushes

Let's see if this causes problems.

The core team is in the bypass list, so they can override this settings, but it is applied for them as well by default.

**Comment by ajeckmans** [comment]
I guess with the changes made we can close this now :)
